alma/controllers/hook/StateHookController.php

```diff
@@ -89,20 +89,14 @@ public function run($params)
 
         switch ($newStatus->id) {
             case $id_state_refund:
-                $msg = "[Alma] Option refund disabled: hook will not be triggered on {$id_state_refund} refund state";
                 if (Settings::isRefundEnabledByState()) {
-                    $msg = "[Alma] Option refund enabled: hook will be triggered on {$id_state_refund} refund state";
                     $this->refund($alma, $id_payment, $order);
                 }
-                Logger::instance()->info($msg);
                 break;
             case $id_state_payment_trigger:
-                $msg = "[Alma] Option PaymentTrigger disabled: hook will not be triggered on {$id_state_payment_trigger} refund state";
                 if (Settings::isPaymentTriggerEnabledByState()) {
-                    $msg = "[Alma] Option PaymentTrigger enabled: hook will be triggered on {$id_state_payment_trigger} refund state";
                     $this->triggerPayment($alma, $id_payment, $order);
                 }
-                Logger::instance()->info($msg);
```

Comment:
Sorry about this removed log... after think, it seems it could be keeped... no ?
Is this logs already serve you to resolve a refund or payment upon trigger issue ?
WDYT @Francois-Gomis @ClaireBonacheraAlma

Reply:
I think maybe just log when the trigger is disabled

(why a switch case for only two conditions and nothing in the default action ?)

Reply:
this info will come with data sharing